components/search/TierToggler.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div>
+    <p class="font-italic">
+      {{ toggleText }}
+      <button
+        class="toggle-button"
+        data-qa="tier toggler"
+        @click="toggleHandler"
+      >
+        {{ !toggle ? button.show : button.hide }}
+      </button>.
+    </p>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'TierToggler',
+
+    props: {
+      toggled: {
+        type: Boolean,
+        default: false
+      },
+      text: {
+        type: Object,
+        required: true
+      },
+      button: {
+        type: Object,
+        required: true
+      },
+      queryKey: {
+        type: String,
+        default: 'tier_zero'
+      }
+    },
+
+    data () {
+      return {
+        toggle: false
+      };
+    },
+
+    computed: {
+      toggleText () {
+        if (this.toggle) {
+          return this.text.hide;
+        }
+        return this.text.show;
+      }
+    },
+
+    mounted () {
+      this.toggle = this.toggled;
+    },
+
+    methods: {
+      toggleHandler () {
+        if (!this.toggle) {
+          this.$emit('click', 'contentTier', ['(*)']);
```

Comment:
leave this to be just `*` without the parentheses. It's not actually helping with the stripping out, it just delays it one time. The problem is that the value for the contentTier facet is not in selectedFacets properly. You can see this if you use the Vue inspect tools on the page components.